XTree/XTree/Extensions/InsettableShape+Ext.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+extension InsettableShape {
```

Comment:
I think it would be better to move this extension into the `DragAndDropView.swift` file and make it private, as it is a specific modification to that view.

And I'm not sure if we can do it without using `#available`. Maybe there is a single solution that works for both versions in a similar way?

Reply:
- I moved it to `DragAndDropView.swift `
- A single solution would be to drop .fill(color) modifier cause it's absent in below 13.0 version. Would you mind that?

Reply:
UPD: so i just removed the extension. And used the 
`.strokeBorder(HierarchicalShapeStyle.tertiary, style: StrokeStyle(dash: [4]))` removed the fill modifier. So visually it looks fine. U might need to check it on Sonoma.
<img width="449" alt="Screenshot 2024-05-15 at 00 02 48" src="https://github.com/swiftyfinch/xtree/assets/36343782/8df03821-dfe0-401b-bf2d-0bb94c50fef6">

Reply:
I will fix this in the main brach